storage/range.go

```diff
@@ -838,46 +864,46 @@ func (r *Range) applyRaftCommand(index uint64, originNodeID multiraft.NodeID, ar
 	return reply.Header().GoError()
 }
 
-// startGossip periodically gossips the cluster ID if it's the first
-// range. This function should only be called if this range replica is
-// the raft leader.
-func (r *Range) startGossip(stopper *util.Stopper) {
-	stopper.RunWorker(func() {
-		r.maybeGossipClusterID()
-		r.maybeGossipFirstRange()
-
-		ticker := time.NewTicker(ttlClusterIDGossip / 2)
-		for {
-			select {
-			case <-ticker.C:
-				if held, expired := r.HasLeaderLease(r.rm.Clock().Now()); held && !expired {
-					r.maybeGossipClusterID()
-					r.maybeGossipFirstRange()
-				}
-			case <-stopper.ShouldStop():
-				return
-			}
-		}
-	})
-}
+// maybeGossipFirstRangeWithLease is periodically called and gossips the
```

Comment:
If we get rid of `Range.start`, let's eliminate this by folding it into `gossipFirstRange`. This all feels rickety. I understand the tests aren't working so well with simplified code. Let's try to fix the tests around that.